plugin.rb

```diff
@@ -21,12 +21,17 @@
 
   module ::Patreon
     PLUGIN_NAME = 'discourse-patreon-creator'.freeze
+    CREATOR_BYPASS_REGEX = /.*(@patreon\.com|patron1@mutations\.ltd)$/
```

Comment:
Sorry to nit-pick, but after re-reading this regexp with fresh eyes, I'm wondering if the `.*` is potentially too lenient of a match? Related, it also may not matter and can be omitted for performance since you're just checking the end of the address value here.